model/FeatureFlag/FeatureFlagFlaHandler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA;
+ *
+ * @author Sergei Mikhailov <sergei.mikhailov@taotesting.com>
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\FeatureFlag;
+
+use oat\tao\model\featureFlag\FeatureFlagCheckerInterface;
+use oat\tao\model\featureFlag\FeatureFlagConfigHandlerInterface;
+
+class FeatureFlagFlaHandler implements FeatureFlagConfigHandlerInterface
+{
+    private FeatureFlagCheckerInterface $featureFlagChecker;
+
+    public function __construct(FeatureFlagCheckerInterface $featureFlagChecker) {
```

Comment:
CI complains https://github.com/oat-sa/extension-tao-itemqti/actions/runs/5443164884/jobs/9899343282?pr=2381
![Screenshot_13](https://github.com/oat-sa/extension-tao-itemqti/assets/38751932/e5721fe1-7fcd-4262-bad8-b03b50eac15a)

Reply:
I'm on it.